docs/config.md

```diff
@@ -192,6 +192,22 @@ Default: `utf-8`
 
 Sets the character encoding to be used when parsing commit messages. [Read more][encoding]
 
+### `template`
+
+Type: `str`
+
+Default: `None` (provided by plugin)
+
+Provide custom changelog jinja template path relative to the current working directory. [Read more][template-customization]
+
+### `extras`
+
+Type: `dict[str, Any]`
+
+Default: `{}`
+
+Provide extra variables to the changelog template. [Read more][template-customization] |
```

Comment:
Is the `|` at the end expected? or is it a typo?

Reply:
No, I think this is a rebase artifact (it was a table by the time I started this PR). There was another one this file which is on `main` too. I added a fix commit for those